Data/Sys/GameSettings/GXX.ini

```diff
@@ -1,8 +1,11 @@
-# GXXE01, GXXP01 - POKeMON XD
+# GXXE01, GXXP01, GXXJ01 - POKeMON XD
 
 [Core]
-# Values set here will override the main Dolphin settings.
+# Prevents crash Greevil's henchman and Zook show up
 CPUThread = False
+# Prevents various invalid read errors, especially during
+# normal (multiplayer) battles when doing multiple battles.
```

Comment:
```suggestion
# normal (multiplayer) battles when doing multiple battles in a row.
```

(Just as a clarification so nobody confuses it for double battles, which are something entirely different)